lib/room.rb

```diff
@@ -1,12 +1,22 @@
-class Rom
+class Room
+  attr_reader :category, :length, :width
 
   def initialize(category, length, width)
     @category = category
-    @length = 10
-    @width = width
+    @length = length.to_i
```

Comment:
not a big deal, but length was already an integer and didn't need to be converted.